### PR TITLE
chore: remove unreferenced devDeps

### PR DESCRIPTION
**Pull request by nschonni**
Few devDeps that don't appear used right now

### CODE REVIEW

**Comment by K-9196** [comment]
IMG_20210414_231434_419
Snapchat

File: package.json [hunk 1]
```diff
@@ -57,11 +57,9 @@
     "endOfLine": "lf"
   },
   "devDependencies": {
-    "@types/brace-expansion": "^1.1.0",
```
Comment (isaacs) [inline package.json:60]: This is necessary for typescript compilation.

**Review by isaacs — CHANGES_REQUESTED** [review]
@types/brace-expansion needs to stay

File: package.json (hunk 1 above)
Comment (nschonni) [reply package.json:60]: I didn't get any warnings/errors from `tsc` as this package is only a subdep of tap, but I added it back anyway

Comment (isaacs) [reply package.json:60]: Oh! lol, I thought this was minimatch, sorry I guess I have too many modules 😅 Yes, that can be removed. I'll get this merged, don't worry about updating a second time.

**Pull request merged**